src/MainWindow.cpp

```diff
@@ -524,7 +508,7 @@ MainWindow::MessageReceived(BMessage* msg)
 			}
 
 			BRect r(Frame());
-			ReconcileWindow* recwin = new ReconcileWindow(BRect(100, 100, 600, 425), acc);
+			ReconcileWindow* recwin = new ReconcileWindow(BRect(100, 100, 700, 425), acc);
```

Comment:
There's the handy "CenterIn()" for this. Like:

`recwin->CenterIn(Frame());`

Reply:
Thanks for the review. I've implemented your recommendations.
I'm no accountant either, maybe someone with experience using financial programs can review at some point, but there are still functional issues that can be worked on in the meantime.